content-fragment-simple-export/src/aem-cf-console-admin-1/web-src/src/components/ExtensionRegistration.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * <license header>
+ */
+
+import { Text } from "@adobe/react-spectrum";
+import { register } from "@adobe/uix-guest";
+import { extensionId } from "./Constants";
+import actions from '../config.json';
+import actionWebInvoke from "../utils.js";
+
+function ExtensionRegistration() {
+  const init = async () => {
+    const guestConnection = await register({
+      id: extensionId,
+      methods: {
+        headerMenu: {
+          getButtons() {
+            return [
+              // YOUR HEADER BUTTONS CODE SHOULD BE HERE
+              {
+                'id': 'cf-export',
+                'label': 'CF Export',
+                'icon': 'OpenIn',
+                async onClick() {
+                  console.log("UIX Export has been pressed.");
+
+                  guestConnection.host.toaster.display({
+                    variant: "positive",
+                    message: "Export generation has started. The download will begin automatically once the process is complete.",
+                    timeout: 3000,
+                  });
+
+                  // const filters = await guestConnection.host.fragmentSelector.getFilters();
```

Comment:
Will be replaced on real filters after merge of https://git.corp.adobe.com/CQ/aem-headless-ui-commons/pull/723